src/main/java/edu/kis/powp/jobs2d/features/DriverFeature.java

```diff
@@ -4,8 +4,9 @@
 import edu.kis.powp.jobs2d.Job2dDriver;
 import edu.kis.powp.jobs2d.drivers.DriverManager;
 import edu.kis.powp.jobs2d.drivers.SelectDriverMenuOptionListener;
+import edu.kis.powp.observer.Subscriber;
 
-public class DriverFeature {
+public class DriverFeature implements Subscriber {
```

Comment:
No, this class should not implement Subscriber. (DriverFeature contains only static elements so there are no instances planned). You already have DriverFeatureChangeObserver. Revert this file.